nexx_integration.module

```diff
@@ -23,7 +23,7 @@ function nexx_integration_library_info_build() {
       'dependencies' => ['core/jquery', 'core/drupal', 'core/backbone'],
       'css' => ['theme' => ['css/styles.css' => []]],
       'js' => [
-        '//require.nexx.cloud/' . $omnia_id => [
+        '//arc.nexx.cloud/sdk/' . $omnia_id . '.play' => [
```

Comment:
This breaks the Burda Style Group instances. I suppose, they have to change some javascript?

Reply:
Any details about the error? It works for MSG and DKR.

Reply:
This breaks because we apply the this pr as a patch on version 1.10, where nexxplay.js is not yet up to date. However I suppose you test this patch against `8.x-1.x` where nexxplay.js is up to date.

Reply:
This PR is based on the latest 8.x-1.x branch, not 1.10

Reply:
Exactly, which is why it breaks. The module is already working for both versions I guess. 

However in 8.x-1.x `nexxplay.js` has been updated to use the new JS API, which is also the reason why you need to update `nexx_integration.module`. However this commit breaks because `1.10` still uses the old API. 

As this PR is not related to the new version (it has been working with the old version) I propose to:

- Create a PR and call it something like `Update nexx_integration module to use new ARC SDK` which can be merged into `8.x-1.x` and revert commit  `741c30d`

This way it is still clean, easy to test and doesn't break Burda Style.

Reply:
so, i will now merfge this fucker, @saviomuc use the dev branch instead of the tag, we will tag if you succeed with the dev branch, OK?

Reply:
ok, we posted at the same time. @saviomuc i still think it is insane to patch a tagged release with a PR that is against the dev branch. I guess there is everything wrong in that approach, that could possibly be wrong :)
It would rather have that tested in the dev branch by you and verify, that everything is fine, if you have the expected codebase. Then we can tag the release, which is overdue anyway

Reply:
I agree that there is better ways than using patches, however it's unfortunately already the case with Burda Style. I am well aware that this leads to many many problems. However this is the current situation and I cannot change a lot. I will test with the dev branch and let you know.

Reply:
Using patches is one thing, but using a dynamic patch on something that is not the base for the patch is really, really bad.
OK, i will merge this now, then you can test.